database/src/seeds/create-user.seed.ts

```diff
@@ -0,0 +1,13 @@
+import { Factory, Seeder } from 'typeorm-seeding'
+import { User } from '../../entity/User'
+
+export class CreateUserSeed implements Seeder {
+  public async run(factory: Factory): Promise<void> {
+    await factory(User)({
+      email: 'peter@lustig.de',
+      firstName: 'Peter',
+      lastName: 'Lustig',
+      username: 'peter',
+    }).create()
+  }
```

Comment:
I'd like to have a file `peter-lustig.user.ts` I can include in my test with `import peter from `@seeds/peter-lustig.user.ts` or similar to be able to seed specific data into the db.